### DISCUSSION TITLE
JBrowse 2 React App Demo：mobx-state-tree.module.js:4272 Error: [mobx-state-tree] You are trying to read or write to an object that is no longer part of a state tree.

### DISCUSSION THREAD

**Discussion by MorganHis** [GMOD/jbrowse-components#4017] [Q&A] [answered]
When I use the "New session" function of the JBrowse 2 React App Demo menu bar, an error occurs: Error: [mobx-state-tree] You are trying to read or write to an object that is no longer part of a state tree. (Object type: 'AnonymousModel', Path upon death: '/session/views/0', Subpath: 'displayedRegions', Action: '/session/views/0.setDefaultSession()'). Either detach nodes first, or don't use objects after removing / replacing them in the tree.
<img width="2559" alt="image" src="https://github.com/GMOD/jbrowse-components/assets/84215074/634a0842-fbd3-4115-ac3b-b2dd5a9fa1c8">

In addition, the "Return to splash screen" function cannot be used because three errors are reported.
<img width="2551" alt="image" src="https://github.com/GMOD/jbrowse-components/assets/84215074/9657d654-3317-4ffc-95bd-9ec041be8fc7">
![image](https://github.com/GMOD/jbrowse-components/assets/84215074/5a7da9a5-c9c4-4605-afa4-8e97a6d3baa3)

**Comment by cmdcolin** [comment, marked as answer]
thanks for reporting. the yellow text is a warning, it is not great but it is not a cause for major concern.

the red text on return to splash screen is an error though, clearly that function also doesn't work! will need to remove that option too, or maybe even allow specifying a custom splash screen...

**Reply by MorganHis** [reply]
Clear reply! It would be very grateful if these two features would also be removed.

**Reply by cmdcolin** [reply]
@MorganHis the "Return to splash screen" removed in v2.7.2 for @jbrowse/react-app. The new session is still there. the warning may persist and we can try to address at some point but AFAIK it is not a  major concern

**Reply by MorganHis** [reply]
@cmdcolin That's awesome! Thanks for removing the 'Return to splash screen.' Best of luck in pinpointing what's causing 'The new session' to not work!
